soql-analyzer/src/test/scala/com/socrata/soql/AnalysisSerializationTest.scala

```diff
@@ -114,7 +114,7 @@ class AnalysisSerializationTest extends FunSpec with MustMatchers {
       testV5Unchained(query)
     }
 
-    it("v4 deserializes to v5 analysis") {
+    ignore("v4 deserializes to v5 analysis") {
```

Comment:
did you mean to leave all these ignored?

Reply:
Different version support is only used for/during deploy.  v4 is behind us.  I leave the template here so that it can be adapted in the future if necessary.